encodings/dict/Cargo.toml

```diff
@@ -30,6 +30,7 @@ workspace = true
 [dev-dependencies]
 criterion = { workspace = true }
 rand = { workspace = true }
+vortex-array = { workspace = true, features = ["test_util"] }
```

Comment:
Oh I liked the `test-harness`! "util" is forbidden!

Reply:
I renamed the extant test_util to test_harness and adjusted all of this PR's features to test_harness.